### PR TITLE
Add homeTask directory files

### PR DESCRIPTION
**Pull request by Sergii112**

### CODE REVIEW

File: homeTask/task2.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
+const avarageGrade = 100
+let studentSuccess
```
Comment (sttaran) [inline homeTask/task2.js]: unused variable. If you do not use it then it should be deleted

File: homeTask/task3_1.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+const selectedNumber = 4
+   console.log(`Multiplication Table ${selectedNumber}:`)
+
+for (let i = 1; i <= 10; i++) {
+    let result = selectedNumber * i
```
Comment (sttaran) [inline homeTask/task3_1.js]: use const

File: homeTask/task3_2.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+const selectedNumber = 2
+    console.log(`Multiplication Table ${selectedNumber}:`)
+
+let i = 1
+    while (i <= 10) {
+        let result = selectedNumber * i
```
Comment (sttaran) [inline homeTask/task3_2.js]: use const

**Review by sttaran — CHANGES_REQUESTED** [review]

File: homeTask/task2.js (hunk 1 above)
Comment (Sergii112) [reply homeTask/task2.js]: Done

File: homeTask/task3_2.js (hunk 3 above)
Comment (Sergii112) [reply homeTask/task3_2.js]: Done

File: homeTask/task3_1.js (hunk 2 above)
Comment (Sergii112) [reply homeTask/task3_1.js]: Done

**Review by Sergii112 — COMMENTED** [review]
task2.js was removed unused const. task3_1/3_2 was use const instead let

**Pull request merged**